Formula/tmux-xpanes.rb

```diff
@@ -0,0 +1,18 @@
+class TmuxXpanes < Formula
+  desc "Ultimate terminal divider powered by tmux"
+  homepage "https://github.com/greymd/tmux-xpanes"
+  url "https://github.com/greymd/tmux-xpanes.git", :tag => "v2.2.1"
+  head "https://github.com/greymd/tmux-xpanes.git", :branch => "master"
+  depends_on "tmux" => :recommended
+
+  def install
+    %w[xpanes tmux-xpanes].each do |cmd|
+      bin.install "bin/#{cmd}"
+      man1.install "man/#{cmd}.1"
+    end
+  end
+
+  test do
+    system "#{bin}/xpanes", "-V"
```

Comment:
You're going to get asked for a functionality test here.

Reply:
Thank you for your feedbacks.
Please wait several days, I will try to work on it.